blocks/rich-text/style.scss

```diff
@@ -37,6 +37,10 @@
 		background: $light-gray-200;
 		font-family: $editor-html-font;
 		font-size: 14px;
+
+		.is-multi-selected & {
```

Comment:
Should the RichText component have awareness of multi-selection, or should this style be applied from the block list stylesheet?

Reply:
I pondered about this a bit, not entirely sure. But this seemed clearer to express.